directory/onion_directory.py

```diff
@@ -3,10 +3,12 @@
 import random               # for random path selection
 import socket               # for TCP communication
 import threading            # for one thread per TCP connection
+from Crypto.Random.Fortuna.FortunaGenerator import AESGenerator
```

Comment:
Fortun and virtuoso are starting to grow on me as var names

Reply:
They grow on you. Just keep in mind `Fortuna` is not my creation :(